e2e/SearchBar.test.js

```diff
@@ -0,0 +1,19 @@
+import Utils from './Utils';
+import TestIDs from '../playground/src/testIDs';
+
+const { elementById, elementByTraits } = Utils;
+
+describe('SearchBar', () => {
+  beforeEach(async () => {
+    await device.launchApp({ newInstance: true });
+    await elementById(TestIDs.OPTIONS_TAB).tap();
+    await elementById(TestIDs.GOTO_SEARCHBAR_SCREEN).tap();
+  });
+
+  it(':ios: show and hide search bar', async () => {
```

Comment:
Lets define the `:ios:` tag on the describe so this entire test isn't executed on Android